### PR TITLE
make: Build a file-based catalog

### PR DESCRIPTION
**Pull request by jhrozek**
Since OLM is deprecating the SQLite based catalog we are currently
using, while at the same time, the SQLite based catalogs have issues
interacting with OpenShift clusters that implement the recent PSA
changes, let's switch to file-based catalog:
    https://olm.operatorframework.io/docs/reference/file-based-catalogs/

Most of the work is done by the "opm render" command except for some
metadata that is stored at catalog/preamble.json, for example the icon.

Jira: CMP-1604

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   [issue](https://github.com/ComplianceAsCode/compliance-operator/issues/8) for
   more details.
 
+- the upstream catalog image is now built using the
```
Comment (mrogers950) [inline CHANGELOG.md:34]: Just capitalize `The` and the rest is good to me.

Comment (jhrozek) [reply CHANGELOG.md]: thanks for the review, done.

**Review by JAORMX — APPROVED** [review]
/lgtm
/approve

Feel free to add the rest of the needed labels.

**Comment by jhrozek** [comment]
Upstream-only task, adding labels

**Pull request merged**